### PR TITLE
el repo enabled toggle added

### PR DESCRIPTION
**Pull request by wimkorevaar**
#### Pull Request (PR) description
Added a toggle to enable the repo in EL

### CODE REVIEW

File: manifests/repo.pp [hunk 1]
```diff
@@ -3,6 +3,7 @@
 class jenkins::repo (
   Stdlib::Httpurl $base_url = 'https://pkg.jenkins.io',
   String $gpg_key_filename = 'jenkins.io.key',
+  Integer $enabled = 1,
```
Comment (ekohl) [inline manifests/repo.pp:6]: Pretty sure `yumrepo` also accepts a `Boolean`, which I think is cleaner.

**Review by ekohl — COMMENTED** [review]
Can this parameter also apply to Debian and Suse?

File: manifests/repo.pp (hunk 1 above)
Comment (wimkorevaar) [reply manifests/repo.pp]: Of course, just a brainfog moment.

Comment (wimkorevaar) [reply manifests/repo.pp]: I'm not that familiar with suse/debian, but I checked the .pp and currently no enable variable was available, but if it is possible to add the toggle to zypprepo or apt::source I won't be against it.

**Comment by wimkorevaar** [comment]
The errors that are generated don't seem to be related to my PR in anyway.

**Review by ekohl — APPROVED** [review]
We can ignore the errors. Thanks!

**Pull request merged**